MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -25,7 +25,11 @@ open class NavigationRouteOptions: RouteOptions {
         shapeFormat = .polyline6
         includesSteps = true
         routeShapeResolution = .full
-        attributeOptions = [.congestionLevel, .expectedTravelTime, .maximumSpeedLimit]
+        if profileIdentifier == .walking {
```

Comment:
Are speed limit annotations available for cycling directions and other profiles that may come up? What if we special-case the `automobile` and `automobileAvoidingTraffic` profiles instead?

Reply:
that's probably the right thing to do for now to avoid potential problems that might come up if other profiles are added later.

Reply:
Turns out requesting `maxspeed` annotations is valid for the `cycling` profile, though it comes back as all `unknown`. It’s only `walking` that throws an error when you request `maxspeed` annotations. So just in case the Directions API lights up speed limits for bikeways in the future – they do exist in real life – we could request those annotations in that case, optimistically. The tradeoff would be a somewhat larger response size.